logo_code42.svg

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The SVGs in your branch look identical to those in `main`. What's the motivation / nature of this change?

Reply:
I accidentally ran an auto formatter that formatted non-py files... I can revert these changes.